Tests/Test_SimulatorInit.c

```diff
@@ -1,6 +1,10 @@
 #include "Simulator.h"
+#include "BSP_CAN.h"
 
 int main(void) {
     Simulator_init();
+    BSP_CAN_Init(NULL, NULL, false);
+    uint8_t data[8] = "NICECOCK";
```

Comment:
Let's consider a more elegant test case. Also, a test for CAN should be made in its own file, with a header that explains how the test should be conducted.

Reply:
whoops thats my bad forgot about that will change and look into making an actual test

Reply:
> Let's consider a more elegant test case. Also, a test for CAN should be made in its own file, with a header that explains how the test should be conducted.

The one thing with doing tests in a .c file is that all our data is being read from the JSON files, so I don't really see how we can specify which JSON file to use.

Reply:
nvm working on it